src/tapyrus-genesis.cpp

```diff
@@ -151,6 +151,11 @@ static int CommandLine()
                 fprintf(stderr, "Error: Aggregate Public Key was invalid.\n");
                 return EXIT_FAILURE;
             }
+            if(!aggpubkey.IsCompressed())
+            {
+                fprintf(stderr, "Error: Uncompressed Aggregate Public Keys are not supported.\n");
```

Comment:
I think `key` is correct, not `keys`, because an aggregate public key is a single key that aggregates multiple public keys. Therefore, this should be changed, including testing code.